1080i/SkinSettings.xml

```diff
@@ -790,6 +790,14 @@
                         <onclick>Skin.ToggleSetting(DisableCenter)</onclick>
                     </control>
 
+                    <control type="radiobutton" id="19709" description="Clearlogo">
+                        <visible>Integer.IsEqual(Window.Property(CurrentFocus),9007)</visible>
+                        <include>Defs_Settings_Button</include>
+                        <label>$LOCALIZE[31170]</label>
+                        <selected>Skin.HasSetting(EnableClearlogo)</selected>
+                        <onclick>Skin.ToggleSetting(EnableClearlogo)</onclick>
+                    </control>
```

Comment:
The setting and label here don't match with the setting and label that you've added.

Reply:
Another error on my part, I pasted the wrong setting, it was a duplicate from clearlogo. Hopefully this correction fixes everything. On another note, I'm sorry for all those changes you see but nothing has been changed. I un-selected them when I committed them but they still showed up. I tried to match it to yours but it keeps reverting back. I did spaces and tabs but it doesn't correct it, what am I missing. Thanks for you time. https://imgur.com/FsL0Yoe

Reply:
Looks like the texteditor you are using is removing trailing whitespace. Which is fine as I should've done that myself anyway.